packages/xy-chart/index.ts

```diff
@@ -69,6 +71,8 @@ export interface XYChartOptions {
   fontFamily: string;
   backgroundColor: string;
   strokeColor: string;
+
+  serverWidth?: number;
```

Comment:
How about call it `chartWidth`?

Reply:
I named it `serverWidth` because it's only used for server side configured width (`mobile`, `laptop`, `desktop`). How 
 about `serverChartWidth`?

Reply:
After more thinking, `chartWidth` is OK because it's more general, and I'll change it.

Reply:
Frond-end could use this field too I think. In the embeded star-history, we could also set a special chart width. 

<img width="1097" alt="image" src="https://user-images.githubusercontent.com/24653555/173728464-93011b62-cb92-43c7-9188-30a2789905d7.png">

And it's better not to have some special target platform fields in `packages/xy-chart`. Make it purer.